types/types.go

```diff
@@ -466,6 +466,18 @@ type ImageInspectInfo struct {
 	Os            string
 	Layers        []string
 	Env           []string
+	LayersDetail  []LayerDetail
+	History       []v1.History
```

Comment:
As this very PR shows, it’s valuable to be able to add extra fields — so a separate `ImageInspectHistory` where we can add fields to would be better than forever tying this to OCIv1, and potentially having to add a `HistoryDetail`.

Reply:
you means I should use another func  ImageInspectHistory to show history?  When I use podman inspect an image, I can see history

Reply:
I mean that the _type_ of the field should be `History []ImageInpectHistory`.